src/tests/JIT/Regression/JitBlue/Runtime_115532/Runtime_115532.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// set DOTNET_TieredCompilation=0
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Numerics;
+using Xunit;
+
+public class Runtime_115532
+{
+    static decimal s_decimal_6 = 27.023809523809523809523809524m;
+    static int s_int_9 = 1;
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private double LeafMethod3()
+    {
+        unchecked
+        {
+            Vector128<double> vdec = Vector128<double>.Zero
+                .WithElement(0, (double)s_decimal_6)
+                .WithElement(1, (double)s_int_9);
+
+            try
+            {
+                vdec = vdec.WithElement(2, 0.0)
+                           .WithElement(3, 0.0);
```

Comment:
Looks like Mono currently asserts for these:
```
16:52:47.954 Running test: JIT/Regression/JitBlue/Runtime_115532/Runtime_115532/Runtime_115532.dll
* Assertion at /Users/runner/work/1/s/src/mono/mono/mini/mini-amd64.c:7783, condition `ins->inst_c0 == 1' not met
```

This should be a relatively simple thing to handle and I'll take a look. CC. @lewing since you've done several of the recent Mono SIMD PRs